webpack.config.js

```diff
@@ -247,7 +247,7 @@ if ( calypsoEnv === 'desktop' ) {
 } else {
 	// jquery is only needed in the build for the desktop app
 	// see electron bug: https://github.com/atom/electron/issues/254
-	webpackConfig.externals.push( 'jquery' );
+	// webpackConfig.externals.push( 'jquery' );
```

Comment:
I'm running into:

```javascript
ReferenceError: jquery is not defined
    at eval (external_"jquery":1)
    at Object.jquery (build.js:14586)
    at __webpack_require__ (manifest.js:788)
    at fn (manifest.js:151)
    at eval (nonce.js:4)
    at Module../node_modules/@wordpress/api-fetch/build-module/middlewares/nonce.js (vendors~gutenberg-editor.js:249)
    at __webpack_require__ (manifest.js:788)
    at fn (manifest.js:151)
    at eval (index.js:11)
    at Module../node_modules/@wordpress/api-fetch/build-module/index.js (vendors~gutenberg-editor.js:213)
```

Reply:
hm. this could be related to the fact that it's an `npm` package depending on `jQuery` and not Calypso code.

@blowery @ockham haven't we run into this before? do we have to try and "transpile the non-transpired" dependency here?

Reply:
That's right, some of the Gutenberg packages (e.g. `@wordpress/api-fetch`) still depend on `jQuery`.

Reply:
I should test, land and publish new version which is `jquery` free later today.

Reply:
I removed this after updating to latest package version.